src/go/rpk/pkg/tuners/iotune/data.go

```diff
@@ -114,6 +115,141 @@ func precompiledData() map[string]map[string]map[string]io {
 			"i3en.metal": {
 				"default": {"", 257024 * 8, 2043674624 * 8, 174080 * 8, 1024458752 * 8},
 			},
+
+			// is4gen values are taken from measurements except where noted.
+			// For reference, the advertised IOPS values also also shown, these are from:
+			// https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/storage-optimized-instances.html
+			// Except for is4gen.8xlarge the alignment between advertised and measured IOPS is very close.
+			// Test results on is4gen show that the BW/IOPS ratio is 8K for reads and 5K for writes.
+			"is4gen.medium": {
+				// advertised   31250             25000
+				"default": {"", 31407, 251665056, 25105, 125274040},
+			},
+
+			"is4gen.large": {
+				// advertised   62500             50000
+				"default": {"", 62804, 503442048, 50214, 251087344},
+			},
+
+			"is4gen.xlarge": {
+				// advertised   125000              100000
+				"default": {"", 125659, 1015560064, 100404, 502173344},
+			},
+
+			"is4gen.2xlarge": {
+				// advertised   250000              200000
+				"default": {"", 251076, 2066053248, 201105, 1012893056},
+			},
+
+			"is4gen.4xlarge": {
+				// advertised   500000              400000
+				"default": {"", 502117, 4132124160, 402148, 2025763456},
+			},
+
+			// The measured values for is4gen.8xlarge broke the expected pattern
+			// and compared to 4xlarge for IOPS were slightly lower (!!) on the read side and
+			// only slightly higher on the write side, rather than the expected 2x
+			// jump. Signs point towards a measurement limitation rather than a real
```

Comment:
For my own understanding, what would be the theoretical impact if it is not a measurement limitation?

Reply:
Well for one it would mean that the instances cannot achieve their advertised performance, which is something to bring up with the cloud vendor I guess. From a practical impact it means that the performance ceiling would be lower than expected on those instance types, but for the most part the gap was small enough that it doesn't necessarily impact performance at the levels expected in our tiers.

Of course I wouldn't expect anyway to simply accept my hand-wavy explanation that this is measurement error and "probably fine" indefinitely :), so here's the follow up: https://github.com/redpanda-data/redpanda/issues/17261